src/utils/date-utils-test.js

```diff
@@ -63,3 +67,26 @@ describe('convertPotentialTimestampToSeconds', () => {
     );
   });
 });
+
+describe('convertPotentialTimestampToMilliseconds', () => {
+  it('converts seconds to milliseconds ', () => {
+    const nowInSeconds = Math.floor(Date.now() * 1000);
+    const nowInMilliseconds = Date.now();
```

Comment:
Could we just call `Date.now()` once in this test? Right now I think there's a small chance the test would fail once in a while